tutor/commands/compose.py

```diff
@@ -245,14 +245,22 @@ def start(
     services: list[str],
 ) -> None:
     command = ["up", "--remove-orphans"]
+    attach = len(services) == 1
     if build:
         command.append("--build")
-    if detach:
+    # We have to run the container in detached mode first to attach to it
+    if detach or attach:
         command.append("-d")
 
     # Start services
     config = tutor_config.load(context.root)
     context.job_runner(config).docker_compose(*command, *services)
+    if attach and not detach:
+        fmt.echo_info(
+            f"""Attaching to service {services[0]}
```

Comment:
In the case that we are running multiple services without `-d`, can you print that `ctrl-z` will exit logs without stopping the services?